File: translations/codenet/uncommented/C2C++/codellama/Code/s859058692.cpp
#include <iostream>
using namespace std;
int main() 
{
    long sum=0;
    int m, n, i, min, max;
    cin >> n;
    for(i=0; i<n; i++){
        cin >> m;
        sum += m;
        if(i == 0){
            min=m;
            max=m;
        }
         
        if(min>m){
            min=m;
        }
        if(max<m){
            max=m;
        }
    }
    cout << min << " " << max << " " << sum << endl;
}

File: translations/codenet/uncommented/C2J/starcoder/Code/s859058692.java
import java.util.*;
public class Main {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        long sum = 0;
        int min = Integer.MAX_VALUE;
        int max = Integer.MIN_VALUE;
        for(int i=0; i<n; i++){
            int m = in.nextInt();
            sum += m;
            if(min > m){
                min = m;
            }
            if(max < m){
                max = m;
            }
        }
        System.out.println(min + " " + max + " " + sum);
    }
}
